projects/go-lib/src/lib/components/go-side-nav/go-nav-group/go-nav-group.component.ts

```diff
@@ -0,0 +1,32 @@
+import { Component, Input, ViewEncapsulation, Output, EventEmitter } from '@angular/core';
+import { NavGroup } from '../nav-group.model';
+import { NavItem } from '../nav-item.model';
+import { GoSideNavService } from '../go-side-nav/go-side-nav.service';
+
+@Component({
+  selector: 'go-nav-group',
+  templateUrl: './go-nav-group.component.html',
+  styleUrls: ['./go-nav-group.component.scss'],
+  encapsulation: ViewEncapsulation.None
+})
+export class GoNavGroupComponent {
```

Comment:
There are a few things in this class that don't have typing. If we want to get the most out of TypeScript we should apply types wherever we can.
```typescript
// is this variable used anywhere?
expanded: boolean = false;

expand(navGroup: NavGroup): void {
```